src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -7488,6 +7488,56 @@ def test_aks_azure_service_mesh_with_pluginca(self, resource_group, resource_gro
             self.is_empty(),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_azure_service_mesh_upgrade_commands(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=79889204&view=results) to validate the change

Reply:
test passed

Reply:
Seems some new test cases were added, queued new [live test validation](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=80989474&view=results).